package.json

```diff
@@ -23,14 +23,14 @@
     "underscore": "1.7.0"
   },
   "devDependencies": {
-    "eslint": "^2.7.0",
+    "eslint": "^4.19.1",
     "grunt": "~0.4.5",
     "grunt-fh-build": "~2.0.0",
     "grunt-contrib-watch": "latest",
-    "grunt-shell": "1.2.1",
-    "istanbul": "^0.4.3",
+    "grunt-shell": "~1.3.1",
+    "istanbul": "^0.4.5",
     "load-grunt-tasks": "^3.5.0",
-    "nock": "0.15.0",
+    "nock": "9.0.17",
```

Comment:
Same with this any breaking changes ?

Reply:
It is a dev lib. It will not affect noting at all. It is used in the tests.